src/main/webapp/dev/js/common/bootboxWrapper.es6

```diff
@@ -28,12 +28,16 @@ function showModalAlert(titleText, messageText, okButtonText, color) {
         buttons: {
             okay: {
                 label: okButtonText || DEFAULT_OK_TEXT,
-                className: `modal-btn-ok btn-${color}` || StatusType.DEFAULT,
+                className: `modal-btn-ok btn-${color || StatusType.DEFAULT}`,
             },
         },
     })
-    // applies bootstrap color to title background
-    .find('.modal-header').addClass(`alert-${color}` || StatusType.DEFAULT);
+    .find('.modal-header')
```

Comment:
Also, @LiHaoTan @whipermr5 What would I pass as the param for the proposed `applyStyleForModal` method? Would the following sample be acceptable:

```JavaScript
function showModalAlert(titleText, messageText, okButtonText, color) {
    bootbox.dialog({
        title: titleText,
        message: messageText,
        buttons: {
            okay: {
                label: okButtonText || DEFAULT_OK_TEXT,
                className: `modal-btn-ok btn-${color || StatusType.DEFAULT}`,
            },
        },
    });
    applyStyleForModal(bootbox.dialog, color);

function applyStyleForModal(modal, color) {
    modal.find('.modal-header')
        // applies bootstrap color to title background
        .addClass(`alert-${color || StatusType.DEFAULT}`)
    .find('.modal-title')
        // adds relevant icon before title
        .addClass(`icon-${color || StatusType.DEFAULT}`);
}
```